gzsitl/gzsitl_plugin.hh

```diff
@@ -54,24 +54,24 @@ class GAZEBO_VISIBLE GZSitlPlugin : public ModelPlugin
 
     // Vehicle Coordinates
     mavlink_vehicles::global_pos_int home_position;
-    math::Pose calculate_pose(mavlink_vehicles::attitude attitude,
+    ignition::math::Pose3d calculate_pose(mavlink_vehicles::attitude attitude,
```

Comment:
I think it would be helpful to add  "using namespace ignition::math", code would look much cleaner.